src/hooks/swap/useSyncTokenDefaults.ts

```diff
@@ -17,6 +19,7 @@ export interface TokenDefaults {
   defaultInputAmount?: number | string
   defaultOutputTokenAddress?: DefaultAddress
   defaultOutputAmount?: number | string
+  defaultChainId?: SupportedChainId
 }
 
 function useDefaultToken(
```

Comment:
I think this can be a fast follow, but there should be a TODO in this code to do it, linked to a jira ticket, ie

    TODO(WEB-1999): useToken will only get a token from the active chain, but should use the passed chainId.